google-spark-operator/operator/base/webhook-svc.yaml

```diff
@@ -0,0 +1,18 @@
+kind: Service
+apiVersion: v1
+metadata:
+  name: spark-operator-webhook
+  namespace: spark-operator
+  labels:
+    app.kubernetes.io/instance: spark-operator
+    app.kubernetes.io/name: spark-operator
+spec:
+  ports:
+    - name: webhook
+      protocol: TCP
+      port: 443
```

Comment:
Why port is 443 and redirects to 8080? Can we at least use a secure port inside the cluster too?

Reply:
It's funny because in fact it's a TLS connection that is opened in the code, even if it's on port 8080. Therefore the certificates that are needed to connect to the Service.
But to avoid confusion I changed it for 8443.